src/Swift.Bindings/src/Marshaler/MarshallingHelpers.cs

```diff
@@ -27,15 +28,20 @@ public static bool MethodRequiresSwiftSelf(MethodEnvironment env)
             return true;
         }
 
-        public static bool StructIsMarshalledAsCSStruct(StructDecl decl)
+        public static bool IsTypeFrozen(TypeRecord typeRecord)
```

Comment:
We might need to have a note somewhere that "Frozen" on the tooling side is not equalt to "Frozen" on swift side (e.g. frozen struct containing a reference to a reference type)

Reply:
Thanks, added as a comment.